go.mod

```diff
@@ -1,55 +1,8 @@
-	github.com/golang/protobuf v0.0.0-20170920220647-130e6b02ab05 // indirect
-	github.com/google/go-querystring v0.0.0-20170111101155-53e6ce116135 // indirect
-	github.com/gopherjs/gopherjs v0.0.0-20181103185306-d547d1d9531e // indirect
-	github.com/hashicorp/errwrap v0.0.0-20141028054710-7554cd9344ce // indirect
-	github.com/hashicorp/go-cleanhttp v0.0.0-20170211013415-3573b8b52aa7 // indirect
-	github.com/hashicorp/go-getter v0.0.0-20170922192948-a686900cb375 // indirect
-	github.com/hashicorp/go-hclog v0.0.0-20171005151751-ca137eb4b438 // indirect
-	github.com/hashicorp/go-multierror v0.0.0-20170622060955-83588e72410a // indirect
-	github.com/hashicorp/go-plugin v0.0.0-20170828024549-3e6d191694b5 // indirect
-	github.com/hashicorp/go-uuid v0.0.0-20160717022140-64130c7a86d7 // indirect
-	github.com/hashicorp/go-version v0.0.0-20170914154128-fc61389e27c7 // indirect
-	github.com/hashicorp/hcl v0.0.0-20170914154624-68e816d1c783 // indirect
-	github.com/hashicorp/hil v0.0.0-20170627220502-fa9f258a9250 // indirect
-	github.com/hashicorp/logutils v1.0.0 // indirect
 	github.com/hashicorp/terraform v0.10.7
-	github.com/hashicorp/yamux v0.0.0-20171005170212-f5742cb6b856 // indirect
-	github.com/jmespath/go-jmespath v0.0.0-20160202185014-0b12d6b521d8 // indirect
-	github.com/jtolds/gls v4.2.1+incompatible // indirect
-	github.com/mattn/go-isatty v0.0.3 // indirect
-	github.com/mitchellh/cli v0.0.0-20170908181043-65fcae5817c8 // indirect
-	github.com/mitchellh/copystructure v0.0.0-20170525013902-d23ffcb85de3 // indirect
-	github.com/mitchellh/go-homedir v0.0.0-20161203194507-b8bc1bf76747 // indirect
-	github.com/mitchellh/go-testing-interface v0.0.0-20171004221916-a61a99592b77 // indirect
-	github.com/mitchellh/hashstructure v0.0.0-20170609045927-2bca23e0e452 // indirect
-	github.com/mitchellh/mapstructure v0.0.0-20170523030023-d0303fe80992 // indirect
-	github.com/mitchellh/reflectwalk v0.0.0-20170726202117-63d60e9d0dbc // indirect
 	github.com/op/go-logging v0.0.0-20160211212156-b2cb9fa56473
-	github.com/pmezard/go-difflib v1.0.0 // indirect
-	github.com/posener/complete v0.0.0-20170908125245-88e59760adad // indirect
-	github.com/satori/go.uuid v1.1.0 // indirect
-	github.com/smartystreets/assertions v0.0.0-20180927180507-b2de0cb4f26d // indirect
-	github.com/smartystreets/goconvey v0.0.0-20181108003508-044398e4856c // indirect
-	github.com/stretchr/testify v1.2.2 // indirect
-	github.com/ulikunitz/xz v0.5.4 // indirect
-	golang.org/x/crypto v0.0.0-20170930174604-9419663f5a44 // indirect
-	golang.org/x/net v0.0.0-20171004034648-a04bdaca5b32 // indirect
-	golang.org/x/sync v0.0.0-20181108010431-42b317875d0f // indirect
-	golang.org/x/sys v0.0.0-20170927054621-314a259e304f // indirect
-	golang.org/x/text v0.0.0-20171006144033-825fc78a2fd6 // indirect
-	google.golang.org/genproto v0.0.0-20171002232614-f676e0f3ac63 // indirect
-	google.golang.org/grpc v1.6.0 // indirect
+	github.com/vmware/terraform-provider-vra7 v0.1.0
```

Comment:
This should be changed: s/vmware/terraform-providers/

Reply:
Thanks 👍